packages/core/src/test/shared/sam/build.test.ts

```diff
@@ -229,6 +232,46 @@ describe('SAM build helper functions', () => {
             assert.deepStrictEqual(quickPick.items, expectedItems)
         })
     })
+
+    describe('resolveBuildFlags', () => {
+        let sandbox: sinon.SinonSandbox
+        let pathAndVersionStub: sinon.SinonStub<any[], any>
+
+        beforeEach(() => {
+            sandbox = sinon.createSandbox()
+        })
+
+        afterEach(() => {
+            sandbox.restore()
+        })
+
+        it('should add --no-use-container if the buildFlags array does not contain --use-container', async () => {
+            const normalVersion = { path: 'file:///path/to/cli', parsedVersion: new SemVer('1.133.0') }
```

Comment:
does it make sense to turn all of this into a function that you can set the parsedVersion and add flags to resolvedBuildFlags and use that function in the 3 tests? The 3 tests look almost the same with minor adjustments

Reply:
Refactered the tests. Thanks